### PR TITLE
fix: missing packages in conman role

### PR DESCRIPTION
**Pull request by bouriquet**
seboolean module requires python selinux libs

### CODE REVIEW

**Review by hmeAtos — DISMISSED** [review]
good catch !
ok for me

File: roles/core/conman/tasks/main.yml [hunk 1]
```diff
@@ -21,11 +21,9 @@
   tags:
     - internal
 
-- name: package █ Install conman and ipmitool
+- name: package █ Install packages
```
Comment (btravouillon) [inline roles/core/conman/tasks/main.yml:24]: Most roles use `{{ rolename_packages_to_install | join(' ') }}`

**Review by btravouillon — COMMENTED** [review]
Indeed, nice catch.
Could you also update `nfs_client` and `openldap_server` that may have the same issue?
```
$ grep -r seboolean: roles/
roles/addons/openldap_server/tasks/main.yml:  seboolean:
roles/core/nfs_client/tasks/main.yml:  seboolean:
roles/core/conman/tasks/main.yml:  seboolean:
```

File: roles/core/conman/tasks/main.yml (hunk 1 above)
Comment (bouriquet) [reply roles/core/conman/tasks/main.yml]: done

**Review by btravouillon — APPROVED** [review]
Looks good to me.

**Review by oxedions — APPROVED** [review]

**Pull request merged**